### DISCUSSION TITLE
Can't download with new update

### DISCUSSION THREAD

**Discussion by tropicalremix** [aclap-dev/video-downloadhelper#872] [Help and Q&A]
### Your question

I was wondering how your supposed to download movies with the new update. I hit download when it shows up in the app but nothing happens

### Addon & CoApp version and details

I was wondering how your supposed to download movies with the new update. I hit download when it shows up in the app but nothing happens

**Comment by GrampaWildWilly** [comment]
You need to post a proper problem report.  The problem creation steps you through the process.  You ignored it.  Go back & do it again, but this time, follow the instructions.  The problem report you have given is completely useless for diagnosing any sort of problem.  You have to provide all the requested information.  Click the Wiki link at the top of every page of this forum.  Down the right side of that page, look for the chapter named "I need help."  Click that.  READ what you find.  FOLLOW the instructions.  I'm not just being pedantic here.  You'll discover that with a proper problem report, you'll get proper support.

**Comment by tropicalremix** [comment]
I am using firefox. When the add-on shows the video with the download button on it. I click  on the download. It just buffers for a few seconds than goes back to the download button again.

**Reply by GrampaWildWilly** [reply]
Since you refuse to actually read the instructions, let me push them into unavoidable view here:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/9e9962ac-2547-4f70-a564-3c61d9b49913)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f8bbaa03-dfd6-4260-9eb0-ac368d209945)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/1d316b3c-3788-4f0a-b3e2-d64474b75186)

That will, as it says, put the important information into your system clipboard.  Then just paste it in here.  That is important information that is necessary if anybody is going to help you.  If you expect any help here, you have to provide that information.  Don't argue with me about it.  Just do it.

Then you must post at least one URL of a web page on which there is a video that is giving you problems.  Don't just tell me a web site.  Don't tell me it happens on all web pages.  I am not experiencing the problem so it is quite obvious that the product works fine for most people.  You have to describe the problem YOU are having in enough detail that somebody else can try to replicate your problem.  I suspect that I will have no problem downloading whatever it is you're talking about.  So I will be sure to illustrate how I managed to successfully get the content.  That should help you do things correctly & also successfully get the content.  If you think it would describe the problem better, post more than one URL.  But it has to be a URL of a web page that contains a video.  It has to be an item that does not require a login, not even a free login.  And it has to be something that doesn't geobl0ck me.

Lots of other users in this forum have followed these instructions & I have been able to help them.  You just have to read some other discussions in here to discover that.  You should be reading other discussions anyway.  There's a fair chance you are encountering a problem somebody else has encountered.

Stop being stubborn.  Follow the procedure.  Following the procedure is not just to satisfy me.  It is to help yourself.  You will be doing this for YOUR benefit, not mine.  Like I say, the product works fine for me.  So help yourself a bit & you will see that I will be able to help you.  But I can't help you until you follow these instructions.  So follow the instructions & let's see where we get.

**Comment by Laidbak300** [comment]
Since the dimwit OP won't follow procedure, I will. I have the same problem. Here's my infodump:

**Comment by Laidbak300** [comment]
version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: "checking"
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "used_sidebar_button": true,
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 3
}

**Comment by Laidbak300** [comment]
If I show Video Downloadhelper in the sidebar, the circle goes around and around forever with the message "Verifying Companion App..." and it stays stuck doing that.

**Reply by GrampaWildWilly** [reply]
Perhaps our interaction will be educational for the original poster & it will prompt him or her to follow the instructions.  I thank you for your proper problem report.  This is something I can work with.

I do not work for VDH.  I am just another user.

I have always found it annoying that the best the problem reporting does is say:

`platform: x86-64 win`

Is that Windows 11?  10?  8?  7?  Heaven forbid, XP, 2K, NT?  I am going to guess that since you are on the woefully out-of-date Firefox 115 that you are stuck on Windows 7.  My sympathies.  I don't think this is going to present a roadblock.  At least, I hope not.  I only about 6 months ago got off of W7 myself after using it for over 10 years & installed W11 on a new computer.

The CoApp for W7 presents a special case.  You need to find where the CoApp is offered, then pick the right CoApp for your environment.  Click the Wiki link you can find at the top of any page in this forum.  That will take you to the closest thing we have to a user manual.  The table of contents for the manual is down the right side of that page.  You would do well to read all of that, if not now, eventually over time.  For today, click the chapter for `CoApp Installation`.  The information there is broken out by operating system.  Under Windows, there are 4 choices, 2 for W7, 2 for newer Windows releases.  In each of those pairs, there's one for the 64-bit flavor & one for the 32-bit flavor.  Pick the one for you & click the button.  This will download an executable that will install the CoApp.

The instructions say to close all browser windows, then run the CoApp installer.  I have found that it is enough to go into about:addons & just disable VDH.  After you do one or the other, execute the installer.  The necessity of closing something explains why you have to do the download & the installation of the CoApp in separate steps.  The installation process is completely automated.  It does give you a chance to pick an alternative location for the CoApp.  My personal taste is to avoid the directory trees that are part of the Windows operating system installation.  So I changed the directory to

`C:\DownloadHelper CoApp`

but that is entirely a matter of taste.  You could even put it on some other disc partition.  The name is not inherently significant.  Call it whatever you like of your choosing or accept the default.  It should work the same either way.

After the installer finishes, you can open up Firefox again or enable VDH again, whichever applies.  Then go into the `More settings` section of the VDH settings & you should see something more or less like this:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f9b7556a-a332-409c-9f70-b5f01b284f62)

That shows the version number of the CoApp & the full file specification, including the directory tree, to the CoApp executable.  Anything different indicates the CoApp installation did not succeed & we would need to try to figure out what went wrong.

So get the CoApp properly installed & let's see how we progress from there.

And try to be nice to fellow users, even when, especially when, they might not deserve it.  Abusive conduct here can result in banishment.  Don't fall into that trap.

**Comment by tropicalremix** [comment]
version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.10","new_version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 4
}

**Comment by tropicalremix** [comment]
i fixed the problem.

**Reply by GrampaWildWilly** [reply]
Be kind to any potential fellow users who might stumble upon this discussion & tell them HOW you fixed your problem.  I note that you apparently have CoApp version 2.0.10, which has been out of date for a rather long time.  The indicated 2.0.19 is the latest.  Did you manage to install that?
